cli/replicasets/edit_topology.go

```diff
@@ -89,7 +109,28 @@ func editInstances(conn net.Conn, opts *EditInstancesOpts) (bool, error) {
 	return true, nil
 }
 
-func serializeEditReplicasetOpts(opts *EditReplicasetOpts) string {
+func serializeEditReplicasetsOpts(opts *EditReplicasetsOpts) (string, error) {
+	replicasetsResults := make([]string, len(*opts))
+	for i, replicasetOpts := range *opts {
+		replicasetRes, err := serializeEditReplicasetOpts(replicasetOpts)
+		if err != nil {
+			return "", project.InternalError("Failed to serialize edit replicaset opts: %s", err)
+		}
+
+		replicasetsResults[i] = replicasetRes
+	}
+
+	res, err := templates.GetTemplatedStr(&editReplicasetsOptsTemplate, map[string]string{
+		"ReplicasetsInput": strings.Join(replicasetsResults, ", "),
+	})
+	if err != nil {
+		return "", project.InternalError("Failed to serialize edit replicasets opts: %s", err)
+	}
+
+	return res, nil
+}
+
+func serializeEditReplicasetOpts(opts *EditReplicasetOpts) (string, error) {
```

Comment:
Do you really need it?

Seems it's the same as `editReplicasetsOptsTemplate` but with single element.

Also `editReplicasetOptsTemplate` and `editReplicasetsOptsTemplate` are quite similar next person who will be change this code could be confused.

`editReplicasetOptsTemplate` + `editReplicasetListOptsTemplate` if you really need it.